packages/ui/components/scheduler/styles/styles.css

```diff
@@ -0,0 +1,17 @@
+.scheduler-wrapper {
+  --gridDefaultSize: 1.75rem;
+  --hoursInDay: 24;
+  --minuteInHour:60;
+  --gridMobileSize: 100vw;
+}
+
+.scheduler-grid-row-template{
+  grid-template-rows: var(--gridMobileSize) repeat(attr(data-gridstoperperday number), var(--gridMobileSize)) auto;
+}
+
+@media (min-width: 640px) {
+  .scheduler-grid-row-template {
+    grid-template-rows: var(--gridDefaultSize) repeat(attr(data-gridstoperperday number), var(--gridDefaultSize)) auto;
```

Comment:
Explaining awful CSS: 

Create a grid with rows - every 1.75 rem and repeat it for the number of grid stops in a day.